cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -85,6 +86,9 @@ the cloud specific control loops shipped with Kubernetes.`,
 
 	fs := cmd.Flags()
 	namedFlagSets := s.Flags()
+	verflag.AddFlags(namedFlagSets.FlagSet("global"))
+	globalflag.AddGlobalFlags(namedFlagSets.FlagSet("global"), cmd.Name())
+	options.AddGenericFlags(namedFlagSets.FlagSet("generic"))
 	for _, f := range namedFlagSets.FlagSets {
```

Comment:
IMO, add the generic like that, no need to add function
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L270